config/application.rb

```diff
@@ -39,6 +39,7 @@ class Application < Rails::Application
       g.helper false
     end
 
+    config.active_record.schema_format = :sql
```

Comment:
Any downsides to this change? Seems like a big deal?

Reply:
So far in my testing it has seemed solid and better actually.  I ran through some db resets to verify things, and reran migrations as well.  With the schema set to SQL I get none of the schema diff thrashing we saw with it set to ruby.